tensorflow_addons/optimizers/tests/stochastic_weight_averaging_test.py

```diff
@@ -80,7 +81,6 @@ def test_assign_batchnorm():
     model.add(tf.keras.layers.Dense(16, activation="relu"))
     model.add(tf.keras.layers.BatchNormalization())
     model.add(tf.keras.layers.Dense(1))
-
     opt = SWA(tf.keras.optimizers.SGD())
     model.compile(optimizer=opt, loss="mean_squared_error")
```

Comment:
The PR is caused by the fact that some callbacks access `model.optimizer.lr`. Does it get tested in this unittest? Maybe have another test instead of this one?

Reply:
Oh, It's my mistake

Reply:
We are evaluating if `lr` is still valid at https://github.com/tensorflow/tensorflow/pull/44295